lemur/plugins/lemur_slack/tests/test_slack.py

```diff
@@ -36,7 +36,7 @@ def get_options():
     return [
         {"name": "interval", "value": 10},
         {"name": "unit", "value": "days"},
-        {"name": "webhook", "value": "https://slack.com/api/api.test"},
+        {"name": "webhook", "value": "https://hooks.slack.com/services/api.test"},
```

Comment:
The fact that this needed to change to a valid value to pass the test is a good sign 🙂